gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.caching=true
+version=0.0.1-SNAPSHOT
```

Comment:
Let's make it a bit more "default"-ish :)

```suggestion
version=0-SNAPSHOT
```

Reply:
tried, doesn't work - not valid for semver parser in pf4j